Xamarin.Forms.Platform.Android/AppCompat/SwitchRenderer.cs

```diff
@@ -8,6 +11,8 @@ namespace Xamarin.Forms.Platform.Android.AppCompat
 	public class SwitchRenderer : ViewRenderer<Switch, SwitchCompat>, CompoundButton.IOnCheckedChangeListener
 	{
 		bool _disposed;
+		ColorStateList defaultOnColor;
```

Comment:
According to [coding stlye](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md),  use _camelCase for internal and private fields.